spec/index.bs

```diff
@@ -7043,6 +7055,17 @@ enum GPUStoreOp {
                 If any of the following conditions are unsatisfied, make |this| [=invalid=] and stop.
                 <div class=validusage>
                     - It is [$valid to draw$] with |this|.
+                    - Let |buffers| be |this|.{{GPURenderEncoderBase/[[pipeline]]}}.{{GPURenderPipeline/[[descriptor]]}}.{{GPURenderPipelineDescriptor/vertex}}.{{GPUVertexState/buffers}}.
+                    - For each {{GPUIndex32}} |slot| `0` to |buffers|.length:
+                        - Let |bufferSize| be |this|.{{GPURenderEncoderBase/[[vertex_buffer_sizes]]}}[|slot|].
+                        - Let |stride| be |buffers|[|slot|].{{GPUVertexBufferLayout/arrayStride}}.
+                        - If |buffers|[|slot|].{{GPUVertexBufferLayout/stepMode}}:
+                            <dl class="switch">
+                                : {{GPUInputStepMode/"vertex"}}
+                                :: |firstVertex| + |vertexCount| &le; |bufferSize| &div; |stride|.
```

Comment:
optional nit: It doesn't make any difference, but perhaps describe this as <code>(firstVertex + vertexCount) \* stride &le; bufferSize</code> to avoid ambiguity between integer division and real/float division? Result is the same either way, but normally we operate on real numbers and it's kind of weird to rely on real number division in an expression about byte offsets.

Same for the other three usages of `&div;`

Reply:
SGTM. Done!